__tests__/app.test.js

```diff
@@ -197,14 +197,57 @@ describe("/api/reviews", () => {
           });
         });
     });
-    test("reviews are sorted by date in descending order", () => {
+    test("reviews are sorted by date in descending order by default", () => {
       return request(app)
         .get("/api/reviews")
         .expect(200)
         .then(({ body }) => {
           expect(body.reviews).toBeSortedBy("created_at", { descending: true });
         });
     });
+    test("reviews can be selected by category", () =>{
+      return request(app)
+      .get("/api/reviews?category=social deduction")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.reviews.length).toBe(11)
+        body.reviews.forEach(review => {
+          expect(review.category).toBe("social deduction")
+        })
+      })
+    })
+    test("reviews can be sorted by any valid column", () => {
```

Comment:
Add another 200 test :)
Status 200, valid category query, but has no reviews responds with an empty array of reviews, e.g. ?category=children's games.